test/utils/test_sdf.py

```diff
@@ -14,10 +14,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ruff: noqa: E402
+
+import urllib.request
+
 import numpy as np
+import pytest
 from pytest_utils import import_or_fail
+from stl import mesh
+
+from modulus.utils.sdf import sdf_to_stl, signed_distance_field
+
+
+@pytest.fixture
+def download_stl(tmp_path):
```

Comment:
Same here, I would separate the tests for the sdf core utility from the tests for this utility.

Reply:
Done!